src/pages/ProductListing.js

```diff
@@ -0,0 +1,60 @@
+import { Navbar } from "../components/Navbar"
+import { Footer } from "../components/Footer"
+import "../styles/pages/productlist.css"
+import { FiltersBar } from "../components/FiltersBar"
+import { ProductCard } from "../components/ProductCard"
```

Comment:
suggestion: you can import all your components in one file and then import the required ones at once